src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -74,6 +76,30 @@ public function setContainer(ContainerInterface $container = null): ?ContainerIn
         return $previous;
     }
 
+    /**
+     * @deprecated since 9.0
```

Comment:
We should add an explanation about the presence of these methods here
```suggestion
     * This method was removed in Symfony 6, for backward compatibility reasons this method is temporarily
     * maintained so the modules can keep using it a little longer. It will be removed in the next major though
     * along with this base controller class.
     *
     * @deprecated since 9.0
```

Reply:
And a similar comment could be added for the three following methods